doc/Extensions/Oxidized.md

```diff
@@ -57,13 +57,13 @@ LibreNMS is able to reload the Oxidized list of nodes, each time a device is add
 To do so, edit the option in Global Settings>External Settings>Oxidized Integration or add the following to your config.php.
 
 ```php
-$config['oxidized']['reload_nodes'] = TRUE;
+$config['oxidized']['reload_nodes'] = true;
```

Comment:
probably out of scope of this PR, but thoughts on removing the config variables or de-emphasizing them in favor of the web config?